build.gradle

```diff
@@ -51,7 +51,7 @@ gradle.startParameter.taskNames.each {
 }
 gradle.startParameter.taskNames = expandedTaskList.flatten()
 
-// Gets a integer command argument, passed with -Pname=x, or the defaut if not provided.
+// Gets a integer command argument, passed with -Pname=x, or the default if not provided.
```

Comment:
did you want to change this to gets 'an' integer command argument while you're there?

Reply:
argh

Reply:
fixed